pkg/api/folder_test.go

```diff
@@ -334,7 +337,7 @@ func TestHTTPServer_FolderMetadata(t *testing.T) {
 		assert.True(t, body.AccessControl[dashboards.ActionFoldersWrite])
 	})
 
-	t.Run("Should attach access control metadata to folder response", func(t *testing.T) {
+	t.Run("Should not attach access control metadata to folder response", func(t *testing.T) {
```

Comment:
This test is testing that metadata doesn't get attached if `?accesscontrol=true` isn't added to the query. So I think it should be `Should not attach...`?

Reply:
Oh so you fixed the description :+1: 
I thought it was a changed you revert in the test but forgot to account for in the description since the test wasn't changed :sweat_smile: